src/Frontend/Path.hs

```diff
@@ -78,17 +81,7 @@ main (Login mb)       root = root </> fromString ("login" <> if fromMaybe True m
 main Logout           root = root </> "logout"
 
 data Space =
-    ListIdeas
-  | ViewIdea (AUID Idea)
-  | EditIdea (AUID Idea)
-  | CommentIdea (AUID Idea)
-  | CreateIdea
-  | ListTopics
-  | ListTopicIdeas (AUID Topic)
-  | ViewTopicIdea (AUID Topic) (AUID Idea)
-  | EditTopicIdea (AUID Topic) (AUID Idea)
-  | CommentTopicIdea (AUID Topic) (AUID Idea)
-  | CreateTopicIdea (AUID Topic)
+    ListTopics
   | ViewTopicIdeasVoting (AUID Topic)
   | ViewTopicIdeasWinning (AUID Topic)
   | ViewTopicDelegations (AUID Topic)
```

Comment:
Nice!  (-:

(I suspect that other `ViewTopicIDeas*` will also move to the idea paths.  That would leave two path constructors under `Space`.)